manifests/overlays/moc/kustomization.yaml

```
@@ -27,3 +27,8 @@ patches:
       version: v1
       kind: Job
       annotationSelector: "operation=chat-notification"
+  - path: put-into-infra-namespace.yaml
+    target: 
+      version: v1
+      kind: EventListener
+      name: aicoe-ci-listener
```

This is not the right way, the above change would apply the aicoe-ci-listener manifest in the infra namespace.
that is not what we want, we want to override the base listener with the moc listener (which you added in this pr)
that can be done with the patch merge strategy, it would patch the base listener with your change.
```suggestion
patchesStrategicMerge:
     -  listener.yaml
```